web/assets/i18n/admin/admin.pot

```diff
@@ -99,11 +99,37 @@ msgctxt "dashboard.settings.tile"
 msgid "Menu tile"
 msgstr ""
 
-#. Textbox header
+#. Background Label Image
 msgctxt "dashboard.settings.tile.background_image_url"
 msgid "Background Image URL"
 msgstr ""
 
+#. Background Label Color
+msgctxt "dashboard.settings.tile.background_color"
+msgid "Background Color"
+msgstr ""
+
+#. Background Label Type
+msgctxt "dashboard.settings.tile.background_type"
+msgid "Background Type"
+msgstr ""
+
+#. Background Option Default Text 
```

Comment:
Maybe "default" could be added as a "common term" as well actually! I'll have a look